sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -420,36 +417,31 @@ export type GetSkillSetOptions = OperationOptions;
 export type GetSynonymMapsOptions = OperationOptions;
 
 // @public
-export interface HighWaterMarkChangeDetectionPolicy {
-    highWaterMarkColumnName: string;
+export type HighWaterMarkChangeDetectionPolicy = DataChangeDetectionPolicyBase & {
     odatatype: "#Microsoft.Azure.Search.HighWaterMarkChangeDetectionPolicy";
-}
+    highWaterMarkColumnName: string;
+};
 
 // @public
-export interface ImageAnalysisSkill {
-    context?: string;
-    defaultLanguageCode?: ImageAnalysisSkillLanguage;
-    description?: string;
-    details?: ImageDetail[];
-    inputs: InputFieldMappingEntry[];
-    name?: string;
+export type ImageAnalysisSkill = SearchIndexerSkillBase & {
     odatatype: "#Microsoft.Skills.Vision.ImageAnalysisSkill";
-    outputs: OutputFieldMappingEntry[];
+    defaultLanguageCode?: ImageAnalysisSkillLanguage;
     visualFeatures?: VisualFeature[];
-}
+    details?: ImageDetail[];
+};
 
 // @public
-export type ImageAnalysisSkillLanguage = 'en' | 'es' | 'ja' | 'pt' | 'zh';
+export type ImageAnalysisSkillLanguage = string;
 
 // @public
-export type ImageDetail = 'celebrities' | 'landmarks';
+export type ImageDetail = string;
 
 // @public
-export type IndexActionType = 'upload' | 'merge' | 'mergeOrUpload' | 'delete';
+export type IndexActionType = "upload" | "merge" | "mergeOrUpload" | "delete";
 
 // @public
 export type IndexDocumentsAction<T> = {
-    __actionType: IndexActionType;
+    actionType: IndexActionType;
```

Comment:
Didn't we name this `__actionType` intentionally so that it wouldn't conflict with a property named `actionType` on the user's document?

Reply:
Yes. Now I have corrected the overrides and the name ```__actionType``` is preserved.